discovery-provider/src/tasks/index_network_peers.py

```diff
@@ -89,7 +89,7 @@ def update_network_peers(self):
             peers_list = list(all_peers)
             # Update creator node url list in IPFS Client
```

Comment:
lets remove this IPFS reference 🙂

Reply:
there's more in index.py i chose to leave out for this PR but i or @theo can remove in a follow up